### PR TITLE
More logging for call caching metrics [BT-322]

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/EngineJobHashingActor.scala [hunk 1]
```diff
@@ -125,6 +125,13 @@ object EngineJobHashingActor {
   case class FileHashes(hashes: Set[HashResult], aggregatedHash: String)
   case class CallCacheHashes(initialHashes: Set[HashResult], aggregatedInitialHash: String, fileHashes: Option[FileHashes]) extends EJHAResponse {
     val hashes = initialHashes ++ fileHashes.map(_.hashes).getOrElse(Set.empty)
+    def aggregatedHashString: String = {
+      val file = fileHashes match {
+        case Some(f) => f.aggregatedHash
+        case None => "None"
+      }
+      s"aggregated hashes: initial = $aggregatedInitialHash, file = $file"
```
Comment (kshakir) [inline engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/EngineJobHashingActor.scala:133]: Using this example from [this travis job](https://app.travis-ci.com/github/broadinstitute/cromwell/jobs/522822618):

```
2021-07-08 11:56:20,976 cromwell-system-akka.dispatchers.engine-dispatcher-28 INFO  - BT-322 c42037b2:globbingscatter.catter2:4:1 cache hit copying success with aggregated hashes: initial = A5A78528BE2830F36B056020A94ABA7C, file = 30994F5C55FC4C7A97ADE2CBBDF20BD5.
```

How does one use these hash values for debugging? I'm assuming `30994F5C55FC4C7A97ADE2CBBDF20BD5` and `A5A78528BE2830F36B056020A94ABA7C` are values that would appear in the database and could be used for further analysis?

**Review by kshakir — APPROVED** [review]
LGTM

File: backend/src/main/scala/cromwell/backend/standard/callcaching/StandardCacheHitCopyingActor.scala [hunk 2]
```diff
@@ -229,7 +229,7 @@ abstract class StandardCacheHitCopyingActor(val standardParams: StandardCacheHit
             case Some(command: IoCopyCommand) =>
               logCacheHitCopyCommand(command)
             case huh =>
-              log.warning(s"BT-322 $jobTag unexpected commandsToWaitFor: $huh")
+              log.warning(s"BT-322 {} unexpected commandsToWaitFor: {}", jobTag, huh)
```
Comment (breilly2) [inline backend/src/main/scala/cromwell/backend/standard/callcaching/StandardCacheHitCopyingActor.scala:232]: Huh... interesting syntax. TIL

**Review by breilly2 — APPROVED** [review]
👍

File: engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/EngineJobHashingActor.scala (hunk 1 above)
Comment (mcovarr) [reply engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/EngineJobHashingActor.scala:133]: Yes that's right. No immediate use for this but we could build log-based metrics that pull out these hashes in labels if we wanted.

**Pull request merged**